test_runner/regress/test_sharding.py

```diff
@@ -712,6 +715,14 @@ def test_sharding_ingest_layer_sizes(
     tenant_id = env.initial_tenant
     timeline_id = env.initial_timeline
 
+    initial_layers_per_shard = {}
```

Comment:
should comment why we are doing this (initdb creates many "Small" image layers).